docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,153 @@
+
+<!-- Should we provide some reading materials on how to tackle this? -->
+
+### Components
+
+#### Zeebe ![active](https://img.shields.io/badge/active-green?style=for-the-badge)
+
+Zeebe, as a workflow engine, is fully **active-active** and replicates data between its brokers. Due to its replication logic, it can handle an entire region failure without data loss. Requiring a proper partition and replication setup. Consider reading through the [cluster technical concept](./../../../components/zeebe/technical-concepts/clustering.md) to learn more about the [raft protocol](<https://en.wikipedia.org/wiki/Raft_(algorithm)>).
+
+#### Elasticsearch ![passive](https://img.shields.io/badge/passive-orange?style=for-the-badge)
+
+We treat Elasticsearch as an **active-passive** component. While it may be possible to deploy it in active-active mode and stretch across regions, it's not officially documented by Elasticsearch. Such a setup generally brings its own challenges along that we can't control and may negatively impact the performance of Zeebe.
+
+We go with the approach of keeping an Elasticsearch per region and populating the data directly from the Zeebe exporter in each region.
+
+#### WebApps ![passive](https://img.shields.io/badge/passive-orange?style=for-the-badge)
+
+WebApps are an **active-passive** component and in its current state also not highly available since we're limited by the included exporters that would cause data issues when running multiple instances at the same time.
+
+Per region, one WebApp instance is required for the exporter to run.
+
+This means that one instance will be actively serving traffic, while the other one is on standby.
+
+##### Operate
+
+In the event of a total primary region loss, the following data will be lost:
+
+- Batch Operations <!-- Do we have a docs link for this? -->
+
+##### Tasklist
+
+In the event of a total primary region loss, the following data will be lost:
+
+- Assignments of tasks
+
+## Requirements
+
+- Camunda 8
+  - Minimum [helm chart version](https://github.com/camunda/camunda-platform-helm) **9.3+**
+  - Minimum component images
+    - Elasticsearch **8.9+**
+    - Operate **8.5+**
+    - Tasklist **8.5+**
+    - Zeebe **8.5+**
+    - Zeebe Gateway **8.5+**
+- Two connected Kubernetes clusters
+- Maximum latency of 100ms between the two Kubernetes clusters
+- Open Ports between the two Kubernetes clusters
+  - **9200** for Elasticsearch since Zeebe has to push data cross region
+  - **26500** for the Zeebe gateway from client/worker communication
+  - **26501** for the Zeebe brokers and Zeebe gateway communication
+  - **26502** for the Zeebe brokers and Zeebe gateway communication
+
+## Limitations
+
+- This is only about a dual-region setup and does not replicate any multi-region setup with more than two Kubernetes clusters
+  - a multi-region setup of 3+ clusters will behave differently due to for example the raft protocol within Zeebe
```

Comment:
```suggestion
  - a multi-region setup of 3+ clusters will behave differently due to, for example, the Raft protocol used by Zeebe
```

(not even sure why we want to mention 3-region here tbh)

Reply:
so people don't get the wrong idea that just because it works for two, it will work for three.

Reply:
removed it altogether